assets/src/dashboard/app/views/templates/detail.js

```diff
@@ -105,8 +114,8 @@ function TemplateDetail() {
   }, [template]);
 
   const activeTemplateIndex = useMemo(
-    () => templates?.findIndex((t) => t.id === template?.id),
-    [template, templates]
+    () => orderedTemplates?.findIndex((t) => t.id === template?.id),
```

Comment:
So the only problem with this is it will return undefined if the `?` falls through. I notice we compare it to 0 on line 148. I think this is fine as long as we understand the data types could be different.

Reply:
ah, so the `?` operator was already part of this when I updated the templates to the orderedTemplates since templates is now an object and the page wants an array to map through. (edited) 
2:04
Given that it'll return undefined I can just swap out the ? for the standard check